CHANGELOG.md

```diff
@@ -1,3 +1,54 @@
+## `5.0.0` Welp Welp Welp
+So it turns out Choo could be radically simplified. We're now comfortably
+sitting at `~4kb`, have removed a whole bunch of words from the API and should
+be a whole lot faster. We've [written about it
+before](https://medium.com/@yoshuawuyts/choo-v5-bc775b007b5e); if you're
+interested we recommend reading through that post.
+
+We're now using an event emitter, mutable state and explicit re-renders. Some
+people might frown at first at the words "mutable state", but if you think it
+through the mental model doesn't change. "State" has always been a concept of
+an object that changes over time; we then render the DOM as a snapshot of that
+state.
+
+What we've done is change the way we mutate that state - we no longer generate
+a ton of expensive intermediate objects to mutate the state, but instead mutate
+the state directly. In turn we've also removed the magic re-rendering and made
+it explicit. This enables people to create tight render loops that can even be
+used in GC constrained environments like games or music production. We think
+this change was well worth it, and will make a lot of sense going forward.
+
+People might also wonder why we've moved away from `flux`/`elm` and are using
+an event emitter now. It turns out that the previous architecture had a lot of
+confusing words that made it harder to learn than it should. It was also not
+possible to react to changes; the thing that changed always had to specify what
+needed to respond to it. By using event emitters we've inverted this, which
```

Comment:
imma pick on words, sorry 🙈 : maybe instead of `inverted this` just say `changed this`